package.json

```diff
@@ -1,6 +1,9 @@
 {
-  "name": "ecqm_module_generator",
-  "version": "1.0.0",
+  "name": "ecq_module_generator",
+  "version": "0.0.0",
```

Comment:
I would suggest the beta version should be `0.0.1`; I'm not sure how things will react to a `0.0.0` version number

Reply:
Yeah I was thinking this too. My plan was to have us do `npm version patch` which will automatically bump the version in package.json and create a 0.0.1 tag. Not sure if this is the best way. Technically we can change it and not use the npm version command and just create the tag manually for the first release. That's probably fine actually.

So in short, Cassidy you can change this to 0.0.1